server/src/test/java/io/crate/integrationtests/LogicalReplicationITest.java

```diff
@@ -437,4 +437,37 @@ public void test_subscription_state_order() throws Exception {
             " JOIN pg_class r ON sr.srrelid = r.oid");
         assertThat(printedTable(res.rows()), is("sub1| t1| r| NULL\n"));
     }
+
+    @Test
+    public void test_write_to_subscribed_table_is_allowed_after_dropping_subscription() throws Exception {
+        executeOnPublisher("CREATE TABLE doc.t1 (id INT) WITH(" + defaultTableSettings() + ")");
+        executeOnPublisher("CREATE TABLE doc.t2 (id INT) WITH(" + defaultTableSettings() + ")");
+        executeOnPublisher("INSERT INTO doc.t1 (id) VALUES (1), (2)");
+        executeOnPublisher("INSERT INTO doc.t2 (id) VALUES (1), (2)");
+
+        // It's important to subscribe to more than 1 table to check
+        // that re-used close/open table logic works with multiple tables
+        createPublication("pub1", false, List.of("doc.t1", "doc.t2"));
+        createSubscription("sub1", "pub1");
+
+        assertThrowsMatches(
+            () -> executeOnSubscriber("INSERT INTO doc.t1 (id) VALUES(3)"),
+            OperationOnInaccessibleRelationException.class,
+            "The relation \"doc.t1\" doesn't support or allow INSERT operations."
+        );
+        assertThrowsMatches(
+            () -> executeOnSubscriber("INSERT INTO doc.t2 (id) VALUES(3)"),
+            OperationOnInaccessibleRelationException.class,
+            "The relation \"doc.t2\" doesn't support or allow INSERT operations."
+        );
+
+        executeOnSubscriber("DROP SUBSCRIPTION sub1 ");
+
+        var response = executeOnSubscriber("INSERT INTO doc.t1 (id) VALUES(3)");
+        assertThat(response.rowCount(), is(1L));
+
+        Thread.sleep(1000);
```

Comment:
Passes with sleep and without it fails with message:
`Expected valid sequence number for replicated op but was unassigned`
I'm wondering if it's not a bug and I need to use better way instead of sleep.